<?xml version="1.0" encoding="utf-8"?>
    <launch>
    <arg name="marker_size" default="40"/>
    <arg name="max_new_marker_error" default="0.08"/>
    <arg name="max_track_error" default="0.2"/>
    <arg name="cam_image_topic" default="/cgo3_camera/image_raw"/>
    <arg name="cam_info_topic" default="/cgo3_camera/camera_info"/>
    <arg name="output_frame" default="/cgo3_camera_link" />
    <arg name="max_frequency" default="10"/>
    <arg name="bundle_files" default="$(find track)/config/landingPad.xml"/> 
    
    <arg name="use_bundles" default="false" />


    <group if="$(arg use_bundles)">
        <node name="ar_track_alvar" pkg="ar_track_alvar" type="findMarkerBundlesNoKinect"
        respawn="false" output="screen"
        args="$(arg marker_size)
        $(arg max_new_marker_error)
        $(arg max_track_error)
        $(arg cam_image_topic)
        $(arg cam_info_topic)
        $(arg output_frame)
	$(arg bundle_files)">	
        </node>
    </group>

    <group unless="$(arg use_bundles)">
        <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect"
        respawn="false" output="screen"
        args="$(arg marker_size)
        $(arg max_new_marker_error)
        $(arg max_track_error)
        $(arg cam_image_topic)
        $(arg cam_info_topic)
        $(arg output_frame)
	$(arg max_frequency)">	
        </node>
    </group>


    <node name="landing_target_pub" pkg="track" type="ar_to_pose.py" respawn="false" output="screen">
        <param name="marker_id" value="ar_marker_105"/>
        <param name="camera_frame" value="cgo3_camera_link"/>
    </node>


</launch>
